jwql/database/database_interface.py

```diff
@@ -423,6 +423,13 @@ class : obj
 NIRCamReadnoiseStats = monitor_orm_factory('nircam_readnoise_stats')
 NIRISSReadnoiseQueryHistory = monitor_orm_factory('niriss_readnoise_query_history')
 NIRISSReadnoiseStats = monitor_orm_factory('niriss_readnoise_stats')
+<<<<<<< Updated upstream
```

Comment:
@mengesser It looks like some merge conflict text got added in here

Reply:
Fixed